ui/position.js

```diff
@@ -142,12 +142,13 @@ $.position = {
 	getWithinInfo: function( element ) {
 		var withinElement = $( element || window ),
 			isWindow = $.isWindow( withinElement[ 0 ] ),
-			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9;
+			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9,
+			hasOffset = element && ( !isWindow && !isDocument );
```

Comment:
Can't this just be reduced to `!isWindow && !isDocument`? It doesn't matter if the user provided the original element.